<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightclubTeleprinter</class>
 <widget class="QMainWindow" name="FightclubTeleprinter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fightclub Teleprinter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="fightclublogolabel">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="apptitle">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;ApplicationName&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="versionlabel">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Version &lt;version&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="qtadvertizing">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Using Qt &lt;QtVersion&gt; on &lt;Platform&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="titlespacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="gplv3logolabel">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/img/gpl-v3-logo.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="titlebttnssepline">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="openClockWindow">
        <property name="text">
         <string>Open Clock Window</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/breeze-icons/window-new-16.svg</normaloff>:/breeze-icons/window-new-16.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="closeAllClockWindows">
        <property name="text">
         <string>Close All Clock Windows</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/breeze-icons/window-close-16.svg</normaloff>:/breeze-icons/window-close-16.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="bttnssepline">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="openAboutDialog">
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/breeze-icons/help-about-16.svg</normaloff>:/breeze-icons/help-about-16.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="openSettingsDialog">
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/breeze-icons/configure-shortcuts-16.svg</normaloff>:/breeze-icons/configure-shortcuts-16.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="bigspacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
